music_assistant/controllers/streams.py

```diff
@@ -928,6 +928,12 @@ async def get_announcement_stream(
             # Finally we also need to make sure we don't make other blocking requests to the TTS
             # data, eg to get the duration (async_parse_tags).
             #
+            if not output_format.codec_type.is_pcm():
+                self.logger.debug(
+                    "Concatenating non-pcm TTS and pre-announce streams, so far players seem to "
+                    "tolerate this, but it might break some player in the future"
+                )
+
```

Comment:
I would just put this as comment in the code and not debug message

Reply:
Done.